playbooks/openstack-delete_caasp.yml

```diff
@@ -1,37 +1,39 @@
 ---
-- hosts: localhost
-  gather_facts: no
-  connection: local
-  vars:
-    stackname_suffix: "caasp"
-  post_tasks:
-    - name: Load generic vars
-      include_vars: "{{ playbook_dir }}/../vars/common-vars.yml"
-    - name: Load openstack vars
-      include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
-
-    - name: Define stackname to create
-      set_fact:
-        stackname: "{{ socok8s_envname }}-{{ stackname_suffix }}"
+#
```

Comment:
Im guessing this is just a drop of openstack-delete_caasp + rename of the caasp4 file. I guess github is not that smart :P

Reply:
Yes, this is the case